xcodeproj/internal/opts.bzl

```diff
@@ -0,0 +1,428 @@
+"""Functions for processing compiler and linker options."""
+
+load("@bazel_tools//tools/cpp:toolchain_utils.bzl", "find_cpp_toolchain")
+load("@build_bazel_rules_swift//swift:swift.bzl", "SwiftInfo")
+
+# C and C++ compiler flags that we don't want to propagate to Xcode.
+# The values are the number of flags to skip, 1 being the flag itself, 2 being
+# another flag right after it, etc.
+_CC_SKIP_COPTS = {
+    "-isysroot": 2,
+    "-mios-simulator-version-min": 1,
+    "-miphoneos-version-min": 1,
+    "-mmacosx-version-min": 1,
+    "-mtvos-simulator-version-min": 1,
+    "-mtvos-version-min": 1,
+    "-mwatchos-simulator-version-min": 1,
+    "-mwatchos-version-min": 1,
+    "-target": 2,
+}
```

Comment:
This feels a little sketch but you know better than I do. Are there cases where a command might be either, and do we need to handle that?

A la:
 --miphoneos-version-min=11.0
 --miphoneos-version-min 11.0

Reply:
The bazel produced ones are always with `=`. I'll check, but I don't think without is supported.

Reply:
Yeah, they only support `=`. I believe that is the case for any of these flags, they only support one way.